Packs/SplunkPy/ReleaseNotes/3_2_3.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### SplunkPy
+
+- Fixed an issue where the *Mirror In* mechanism has stopped when ***get-modified-remote-data*** command failed on error.
```

Comment:
```suggestion
Fixed an issue where the *Mirror In* mechanism has stopped when the ***get-modified-remote-data*** command failed on error.
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.